src/test/java/pl/stormit/eduquiz/game/PlayGameIntegrationTest.java

```diff
@@ -67,8 +67,7 @@ void shouldCreateGameCorrectly() {
         //when
```

Comment:
Please correct each section: given/when/then ->
from:
//given
//when
//then

into:
// given
// when
// then
*Tomek pointed this out in code review